wp-content/themes/citylimits/blocks/citylimits-custom-donate/style.css

```diff
@@ -0,0 +1,55 @@
+/**
+ * The following styles get applied both on the front of your site and in the editor.
+ *
+ * Note that 0.75rem here is equal to 0.5 * @measure in the LESS files in this site.
+ */
+
+.wp-block-citylimits-citylimits-custom-donate {
+  border-top: 1px solid black;
+  border-bottom: 1px solid white;
+  background-color: black;
+  color: #fff;
+  padding: 0;
+  background-image: url( '../../img/footer-skyline.png' );
+  background-repeat: no-repeat;
+  background-position: bottom center;
+  background-size: auto 50px;
+}
+.wp-block-citylimits-citylimits-custom-donate h3.widgettitle {
+  color: white;
+  font-size: 20px;
+}
+.wp-block-citylimits-citylimits-custom-donate .border {
+  margin: 0.75rem;
+  border: 1px solid white;
+  padding: 1.5rem 1.5rem 50px;
+  clear: both;
+}
+.wp-block-citylimits-citylimits-custom-donate .border {
+  max-width: 20em;
```

Comment:
@benlk The `max-width` is causing issues now on the default variant with no alignment :/

![Screen Shot 2019-11-11 at 4 39 57 PM](https://user-images.githubusercontent.com/18353636/68623232-026afb00-04a2-11ea-9fa4-b5c987ec7490.png)

Reply:
Give https://github.com/INN/umbrella-citylimits/pull/104/commits/c3fe29224c6802136eec08908bafff254084a0a1 a try?

Reply:
Looks great!